modules/sdk/project-templates/project-templates-ext/bnd.bnd

```diff
@@ -0,0 +1,9 @@
+Bundle-Description: Creates a Liferay ext module project that override existing Liferay modules.
```

Comment:
`Creates a Liferay Modules Ext project that overrides an existing Liferay module.`